.tool-versions

```diff
@@ -0,0 +1,2 @@
+ruby 2.6.8
```

Comment:
Sounds good to me! Do you know what those files are for? I've always disregarded them tbh 🤷🏻‍♂️

Reply:
Those `.sample` files are usually there so developers can kick start projects.
Since we tell `git` to ignore version manager files such as `.tool-versions` and `.ruby-version`, we put samples in the repo instead.

Reply:
Following Aleksander's preference I'm gonna leave the `.sample` file and the version files ignored.